go/exorelay/exorelay_test.go

```diff
@@ -142,7 +142,7 @@ func TestMain(m *testing.M) {
 	status := godog.RunWithOptions("godogs", func(s *godog.Suite) {
 		FeatureContext(s)
 	}, godog.Options{
-		Format:        "pretty",
+		Format:        "progress",
```

Comment:
Can we at least make it so that it uses `progress` when running all tests, but uses `pretty` when running a single test? The check for that condition is on line 137.